File: fixtures/python/hello.py
"""minimal fixture for py-direct verification."""
from __future__ import annotations


def greet(name: str) -> str:
    return f"hello, {name}"


def add(a: int, b: int) -> int:
    return a + b


class Counter:
    def __init__(self, start: int = 0) -> None:
        self.value = start

    def increment(self, by: int = 1) -> int:
        self.value += by
        return self.value


if __name__ == "__main__":
    print(greet("world"))
    print(add(1, 2))
    c = Counter()
    print(c.increment(5))

File: fixtures/csharp/hello.cs
namespace Fixture;

public static class Greeter
{
    public static string Greet(string name) => $"hello, {name}";

    public static int Add(int a, int b) => a + b;
}

public class Counter
{
    private int _value;

    public Counter(int start = 0) => _value = start;

    public int Increment(int by = 1)
    {
        _value += by;
        return _value;
    }
}
